### PR TITLE
changed comment to discourage tinkering w/ theme, etc.

### PR DESCRIPTION
**Pull request by drscotthawley**
/preview
Original wording made it seem like changes to theme & more were optional, but a stronger statement is that they are not supported at all.

### CODE REVIEW

**Comment by drscotthawley** [comment]
the deploy failed because I didn't set this up as a full blog site -- there are no deploy keys for this repo because I'm only modifying the source code.

**Comment by drscotthawley** [comment]
I have the keys set up, but the usual button to re-run is missing from the Actions page.  Hmm.

**Comment by drscotthawley** [comment]
Following instructions on contributing, re. testing locally using [dev tools](https://github.com/drscotthawley/fastpages/blob/master/_dev_tools/README.md):

```
$ cd fastpages/_dev_tools/
$ ls
convert-docker-compose.yaml  docker-compose.yaml  Makefile  README.md
$ make build
docker stop fastpages-dev || true; docker rm fastpages-dev || true;
Error response from daemon: No such container: fastpages-dev
Error: No such container: fastpages-dev
docker-compose -f convert-docker-compose.yaml  build --no-cache
make: docker-compose: Command not found
make: *** [Makefile:8: build] Error 127
$
```
Docker is installed on my system and the daemon is running, and I've used it quite a bit. 
But this command "docker-compose" is unknown.  It is not listed as a dependency in the docs.  
?

File: _config.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ use_math:
 # Add your Google Analytics ID here if you have one and want to use it
 google_analytics:
 
-# Everything below here can be ignored, unless you want to change the theme entirely
+# Everything below here should be left alone. Modifications may break fastpages
```
Comment (hamelsmu) [inline _config.yml]: Hmmm you could change the plugins but not the theme so perhaps this can be worded more carefully ... will think about this further

**Pull request merged**